VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.scss

```diff
@@ -19,13 +19,26 @@
       margin-right: 0.5em;
     }
   }
+  .participant-group-row {
+    display: flex;
+    border-bottom: 0.5px rgba(255, 255, 255, 0.2);
+    padding-top: 1em;
+    padding-bottom: 0.2em;

```

Comment:
This is the same as .participant-row except the padding-bottom. Should probably just have padding-bottom and add this as an additional class to those rows.

Reply:
it differs only in the padding bottom & border 
I will assign the class to the required participants only